### PR TITLE
Added Hacktoberfest presences

### PR DESCRIPTION
**Pull request by Xyszx**
image
image
image
image
image

### CODE REVIEW

File: websites/H/Hacktoberfest/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "author": {
+      "name": "Xyszx",
+      "id": "593774699654283265"
+    },
+    "service": "Hacktoberfest",
+    "description": {
+      "en": "Hacktoberfest® is open to everyone in our global community. Whether you’re a developer, student learning to code, event host, or company of any size, you can help drive growth of open source and make positive contributions to an ever-growing community. All backgrounds and skill levels are encouraged to complete the challenge. "
+    },
+    "url": "hacktoberfest.digitalocean.com",
+    "version": "0.8.8",
```
Comment (ririxichan) [inline websites/H/Hacktoberfest/dist/metadata.json:11]: version should be 1.0.0 on release

File: websites/H/Hacktoberfest/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "logo": "https://imgur.com/W4srxuR.png",
+    "thumbnail": "https://imgur.com/sTgmNCW.jpg",
```
Comment (ririxichan) [inline websites/H/Hacktoberfest/dist/metadata.json:13]: it should be i.imgur.com

File: websites/H/Hacktoberfest/tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+    "extends": "../../../tsconfig.json",
+    "compilerOptions": {
+      "outDir": "./dist/"
+      
+    }
+  }
+
```
Comment (ririxichan) [inline websites/H/Hacktoberfest/tsconfig.json:8]: fix spacing

**Review by ririxichan — CHANGES_REQUESTED** [review]

File: websites/H/Hacktoberfest/tsconfig.json (hunk 3 above)
Comment (Xyszx) [reply websites/H/Hacktoberfest/tsconfig.json]: Done all

**Review by Xyszx — COMMENTED** [review]
All done.

File: websites/H/Hacktoberfest/presence.ts [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+      presenceData.details = `Viewing Page:`;
+      presenceData.state = `Event Organizer Kit`;
+    }
+    else if (document.location.pathname.includes("/updates")) {
+      presenceData.details = `Reading Page:`;
+      presenceData.state = `Hacktoberfest Update`;
+    }
+    else if (document.location.pathname.includes("/login")) {
+      presenceData.details = `Viewing Page:`;
+      presenceData.state = `Login`;
```
Comment (ririxichan) [inline websites/H/Hacktoberfest/presence.ts:36]: change ` to " like in /faq ``if``

**Review by ririxichan — CHANGES_REQUESTED** [review]

**Comment by Xyszx** [comment]
Done.

File: websites/H/Hacktoberfest/dist/metadata.json [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    },
+    "service": "Hacktoberfest",
+    "description": {
+      "en": "Hacktoberfest® is open to everyone in our global community. Whether you’re a developer, student learning to code, event host, or company of any size, you can help drive growth of open source and make positive contributions to an ever-growing community. All backgrounds and skill levels are encouraged to complete the challenge. "
+    },
+    "url": "hacktoberfest.digitalocean.com",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/W4srxuR.png",
+    "thumbnail": "https://i.imgur.com/sTgmNCW.jpg",
+    "color": "#000000",
```
Comment (Bas950) [inline websites/H/Hacktoberfest/dist/metadata.json:14]: is that the color hex you want to use? I would use like the blue background color of the logo or something like that.

Comment (Xyszx) [reply websites/H/Hacktoberfest/dist/metadata.json:14]: Wait, i change.

**Review by Xyszx — COMMENTED** [review]
Done.

File: websites/H/Hacktoberfest/dist/metadata.json [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+    "author": {
+      "name": "Xyszx",
+      "id": "593774699654283265"
+    },
+    "service": "Hacktoberfest",
+    "description": {
+      "en": "Hacktoberfest® is open to everyone in our global community. Whether you’re a developer, student learning to code, event host, or company of any size, you can help drive growth of open source and make positive contributions to an ever-growing community. All backgrounds and skill levels are encouraged to complete the challenge. "
+    },
+    "url": "hacktoberfest.digitalocean.com",
+    "version": "1.0.3",
```
Comment (ririxichan) [inline websites/H/Hacktoberfest/dist/metadata.json:11]: Version still should be 1.0.0 since you are releasing presence first time.

**Review by ririxichan — CHANGES_REQUESTED** [review]

File: websites/H/Hacktoberfest/dist/metadata.json (hunk 6 above)
Comment (Xyszx) [reply websites/H/Hacktoberfest/dist/metadata.json]: Okay, done.

**Review by ririxichan — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**